apps/api/middleware/jwt_auth.py

```
@@ -47,18 +51,61 @@ class AuthClaims(TypedDict, total=False):
+        RuntimeError: If no secret found, too short, or has insufficient entropy.
+    """
+    auth_secret = os.getenv(AUTH_SECRET_ENV_VAR) or os.getenv("BETTER_AUTH_SECRET")
+
     if not auth_secret:
-        logger.error("AUTH_SECRET environment variable is required for JWT authentication")
-        raise RuntimeError("AUTH_SECRET environment variable is required for JWT authentication")
+        logger.error("AUTH_SECRET or BETTER_AUTH_SECRET required for JWT authentication")
+        raise RuntimeError(
+            "AUTH_SECRET (or BETTER_AUTH_SECRET) environment variable must be set. "
+            "Generate with: python -c 'import secrets; print(secrets.token_urlsafe(32))'"
+        )
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)